.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -9,39 +9,39 @@ In order to ease downstream packaging of Rust binaries, an archive of vendored d
 
 This guide requires:
 
- * a web browser (and network connectivity)
+ * A web browser (and network connectivity)
  * `git`
- * GPG setup and personal key for signing
+ * [GPG setup][GPG setup] and personal key for signing git commits
  * `cargo` (suggested: latest stable toolchain from [rustup][rustup])
  * `cargo-release` (suggested: `cargo install -f cargo-release`)
  * A verified account on crates.io
  * Write access to this GitHub project
+ * Membership in the [Fedora CoreOS Crates Owners group](https://github.com/orgs/coreos/teams/fedora-coreos-crates-owners/members)
 
 ## Release checklist
 
-These steps show how to release version `x.y.z` on the `origin` remote (this can be checked via `git remote -av`).
+These steps show how to release version `x.y.z` on the `upstream` remote (this can be checked via `git remote -av`).
```

Comment:
Let's keep `origin` here, we have the same setup/checklist in all the other repos.

Reply:
ah I didn't know this was a common one. I changed it to `upstream` since I thought `origin` would usually be the name of one's own fork (hence the warning).